DuckDuckGo/TabViewControllerBrowsingMenuExtension.swift

```diff
@@ -87,7 +87,7 @@ extension TabViewController {
     
     private func buildFindInPageAction(forLink link: Link) -> UIAlertAction? {
         return UIAlertAction(title: UserText.findInPage, style: .default) { [weak self] _ in
-            Pixel.fire(pixel: .browsingMenuFindInPage)
```

Comment:
I don't think this was intentional change.

Reply:
you are correct.. it wasn't..